server/sonar-server/src/main/java/org/sonar/server/ui/PageDecorations.java

```diff
@@ -17,18 +17,20 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.server.ui;
 
-import org.sonar.api.server.ServerSide;
-import org.sonar.api.web.PageDecoration;
+package org.sonar.server.ui;
 
 import java.util.Collections;
 import java.util.List;
+import org.sonar.api.server.ServerSide;
+import org.sonar.api.web.PageDecoration;
 
 /**
  * @since 3.3
+ * @deprecated since 6.3 see {@link org.sonar.api.web.page.PageDefinition}. This class has not effect.
  */
 @ServerSide
+@Deprecated
 public final class PageDecorations {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)